llvm/lib/ProfileData/InstrProf.cpp

```diff
@@ -1704,19 +1685,17 @@ size_t Header::size() const {
                   "been added to the header, if not add a case statement to "
                   "fall through to the latest version.");
```

Comment:
Update the comment and assert message?

Reply:
done. I also updated the assert message in `static_assert(std::is_standard_layout_v<Header>`.